music/demos/visualizer.ts

```diff
@@ -24,11 +24,19 @@ const MIDI_URL = './melody.mid';
 
 let canvasVisualizer: mm.PianoRollCanvasVisualizer;
 let svgVisualizer: mm.PianoRollSVGVisualizer;
+let staffVisualizer: mm.StaffSVGVisualizer;
+
+let staffConfig: mm.StaffSVGVisualizerConfig = {
```

Comment:
can these be defaults, like we have defaults in the PianoRollVisualizer?

Reply:
Ok